src/quattor-client/pom.xml

```diff
@@ -0,0 +1,127 @@
+    <version>1.43</version>
+  </parent>
+
+  <licenses>
+    <license>
+      <name>Apache 2 License</name>
+      <url>http://www.opensource.org/licenses/apache2.0</url>
+      <comments>Copyright (c) Responsible Organization</comments>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>James Adams</name>
+      <email>james.adams@stfc.ac.uk</email>
+    </developer>
+  </developers>
+
+  <build>
+    <plugins>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>tarballs</id>
+            <phase/>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>pod2man</id>
+            <phase/>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-resources-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>filter-sources</id>
+            <phase>process-sources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration combine.self="override">
+              <outputDirectory>${project.build.directory}</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>src</directory>
+                  <includes>
+                    <include>*.repo</include>
+                  </includes>
+                  <filtering>true</filtering>
+                </resource>
+              </resources>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>rpm-maven-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <summary>Quattor client meta-package</summary>
+          <name>${project.artifactId}</name>
+          <url>https://github.com/quattor/release</url>
+          <requires>
+            <require>ncm-spma</require>
+            <require>ncm-cdispd</require>
+            <require>ccm</require>
+            <require>cdp-listend</require>
+            <require>ncm-query</require>
+            <require>ncm-ncd</require>
+            <!-- The following should really be dependencies of other packages -->
+            <require>perl-AppConfig</require>
+            <require>yum-plugin-versionlock</require>
+            <require>yum-priorities</require> <!-- This was renamed to yum-plugin-priorities in xL6, but this still works for for xL[5-7] -->
+          </requires>
+          <defaultDirmode>755</defaultDirmode>
+          <mappings>
+            <mapping>
+              <directory>/etc/quattor/</directory>
```

Comment:
Maybe it should be /etc/quattor-release with a version number in it, I don't know.